tailwind.config.js

```diff
@@ -16,6 +16,53 @@ module.exports = {
     },
     fontFamily: {
       body: ["Outfit"]
+    },
+    height: {
+      "cg-s":"50rem",
+      "cg-l":"64.208rem"
+    },
+    maxWidth:{
+      "cg-l":"94.625rem",
+      "cg-s":"6.088rem",
+      "cg-l-herodiv1":"14.75rem",
+      "cg-s-herodiv2":"18.727rem",
+      "cg-l-herodiv2":"45.375rem",
+      "cg-l-herodiv3":"45.094rem",
+      "cg-s-herodiv3":"18.611rem",
+      "cg-l-herodiv4":"44rem",
+      "cg-s-herodiv4":"16.688rem",
+      "cg-l-herodiv5":"32.75rem",
+      "cg-s-herodiv5":"13.517rem",
+      "cg-s-herodiv6":"10.473rem",
+      "cg-l-herodiv6":"25.375rem",
+      "cg-md-herodiv5":"28.643rem",
+      "cg-md-herodiv6":"22.193rem",
+      "cg-md-herodiv4":"35.363rem",
+      "cg-md-herodiv3":"39.439rem",
+      "cg-md-herodiv2":"39.684rem",
+      "cg-md-herodiv1":"12.9rem"
+    },
+    maxHeight:{
+      "cg-l-herodiv1":"23.563rem",
+      "cg-l-herodiv2":"54.593rem",
```

Comment:
If you are missing one pixel on defined ones it's okay you can leave it, and use the defined one this will help you to minimize the number of customized Tailwind CSS classes, and again as many values are in numbers here you can use numbers instead of names.  If you check here on the official documentation you will see that the values of max--height are numbers so it is better to use numbers instead of strings. [max-height](https://tailwindcss.com/docs/max-height)